JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -515,384 +651,534 @@
+    </owl:Class>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/SpecificEnergyConsumption -->
+
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/SpecificEnergyConsumption">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/FactoryProperty"/>
+        <rdfs:comment>Specific energy consumption of a factory. It refers to the quantity of energy used per unit mass of product or unit of sales revenue.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
         <rdfs:label>Specific Energy Consumption</rdfs:label>
     </owl:Class>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#StorageCapacity -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/StorageCapacity -->
+
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/StorageCapacity"/>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/ThermalEfficiency -->
+
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/ThermalEfficiency">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/FactoryProperty"/>
+        <rdfs:comment>The proportion of input energy converted to useful work.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Thermal Efficiency</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/UtilizationRate -->
+
 
-    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany#StorageCapacity"/>
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/UtilizationRate">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/DataCentreProperty"/>
+        <rdfs:comment>The typical utilization rate of a data centre.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Utilization Rate</rdfs:label>
+    </owl:Class>
     
 
 
     <!-- http://www.theworldavatar.com/kg/ontopowsys/powsysfunction/PowerConsumption -->
 
+
     <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontopowsys/powsysfunction/PowerConsumption">
         <rdfs:subClassOf rdf:resource="http://www.ontology-of-units-of-measure.org/resource/om-2/Quantity"/>
         <rdfs:comment>The utilisation of power to satisfy needs or in manufacturing, e.g., chemical manufacturing.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/ontology/ontopowsys/PowSysFunction.owl</rdfs:isDefinedBy>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/ontology/ontopowsys/PowSysFunction.owl"/>
         <rdfs:label>Power Consumption</rdfs:label>
     </owl:Class>
     
 
 
     <!-- http://www.theworldavatar.com/ontology/ontocape/chemicalprocesssystem/cpsrealization/plant/PlantItem -->
 
+
```

Comment:
Same comment as before - why do the following concepts not have any additional properties other than the defined IRI?

Reply:
I have ensured that all the classes and relations defined in OntoCompany have appropriate explanations and properties.